jupyter-book/deconvolution/deconv_covid_MuSiC_v0.ipynb

```diff
@@ -0,0 +1,2416 @@
+{
```

Comment:
<p><code>sc.pp.highly_variable_genes</code> with flavour <code>cell_ranger</code> requires logarithmized data, right? so first log transform and then select genes, otherwise this contradicts the requirements of the scanpy function. or add a sentence why you use it this way</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Hana? Did you do that for a reason? I think that we can replace the order.</p>

Reply:
<p>adata_log is only for visualisation. We can use another flavor that doesn't require log transformation</p>